GraphicsForAxisEvents/html/index.js

```diff
@@ -0,0 +1,130 @@
+const eventForms = [];
+const buttons = [];
+
+$(document).ready(function () {
+    $.get('/local/camscripter/package/settings.cgi?package_name=graphics_for_axix_events&action=get', function (settings) {
+        $('#scIP').val(settings.sourceCamera.IP);
+        $('#scPort').val(settings.sourceCamera.port);
+        $('#scUser').val(settings.sourceCamera.user);
+        $('#scPassword').val(settings.sourceCamera.password);
+
+        $('#tcIP').val(settings.targetCamera.IP);
+        $('#tcPort').val(settings.targetCamera.port);
+        $('#tcUser').val(settings.targetCamera.user);
+        $('#tcPassword').val(settings.targetCamera.password);
+
+        for (let event of settings.events) {
+            addEventFormHTML(event);
+        }
+    });
+
+    $(".form-control").change(inputChanged);
+    $("#addNewEvent").click(addNewEvent_Click);
+    $(".myForm").submit(function () {
+        return false;
+    });
+});
+
+function addNewEvent_Click() {
```

Comment:
asi bych radsi zas pouzil CamelCase at je to konzistentni..

Reply:
To jsem jenom zvyklý z Windows Forms...

Reply:
!